aztec_gddt/plot_tools.py

```diff
@@ -53,18 +53,18 @@
 
 
 def extract_df(df_to_use: DataFrame,
-               params_to_use: List[str] = [],
-               trajectory_kpis: Dict[str, Callable] = {},
-               agg_columns: List[str] = [],
-               cols_to_drop: List[str] = []) -> DataFrame:
+               params_to_use: List[str] = None,
```

Comment:
We should change the type hint to be Optional[List[str]] if we want to use `None` as a value